x-pack/plugins/siem/public/components/navigation/tab_navigation/index.tsx

```diff
@@ -7,6 +7,7 @@ import { EuiTab, EuiTabs } from '@elastic/eui';
 import * as React from 'react';
 
 import { getHostsUrl, getNetworkUrl, getOverviewUrl, getTimelinesUrl } from '../../link_to';
+import { trackUiAction } from '../../../lib/track_usage';
```

Comment:
I would keep things consistent with what the recommended API usage is.

It looks to be:

```ts
import { trackUiAction as track } from '../../lib/track_usage';
```

So I would import it just like that and then do that here. Typically we avoid the `as` syntax when importing but it looked like the recommended usage was using the `as` so I would stick to it for now and use it uniformly.